pkg/services/anonymous/anonimpl/impl.go

```diff
@@ -81,6 +85,11 @@ func (a *AnonDeviceService) usageStatFn(ctx context.Context) (map[string]any, er
 }
 
 func (a *AnonDeviceService) tagDeviceUI(ctx context.Context, device *anonstore.Device) error {
+	err := a.validator.Validate(ctx)
```

Comment:
@leandro-deveikis  nice work.

i find that validator/validate is a bit abstract, it is enforcing a limit. so should the interface be `CheckLimit` .

to me i read it as, validate what? :honk_think:

Reply:
fixed in last commit! thanks for the feedback!